tests/auxtel/test_latiss_wep_align.py

```diff
@@ -0,0 +1,676 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import random
+import unittest
+import asyncio
+import numpy as np
+import warnings
+import os
+
+import pytest
+
+try:
+    # TODO: (DM-24904) Remove this try/except clause when WEP is adopted
+    from lsst.ts import wep
+
+    WEP_AVAILABLE = True
+except ImportError:
+    WEP_AVAILABLE = False
+    warnings.warn("Could not import cwfs package. Most tests will be skipped.")
+
+from lsst.ts import salobj
+from lsst.ts import standardscripts
+from lsst.ts import externalscripts
+from lsst.ts.externalscripts.auxtel import LatissWEPAlign
+import lsst.daf.butler as dafButler
+from lsst.utils import getPackageDir
+import logging
+
+# Make matplotlib less chatty
+logging.getLogger("matplotlib").setLevel(logging.WARNING)
+# Make obslsst translators less chatty
+logging.getLogger("lsst.obs.lsst.translators").setLevel(logging.WARNING)
+logging.getLogger("astro_metadata_translator").setLevel(logging.WARNING)
+logging.getLogger("flake8.style_guide").setLevel(logging.WARNING)
+
+
+logger = logging.getLogger(__name__)
+logger.propagate = True
+
+random.seed(47)  # for set_random_lsst_dds_partition_prefix
+# Check to see if the test data is accessible for testing at NCSA
+# Depending upon how we load test data this will change
+# for now just checking if butler is instantiated with
+# the default path used on the summit and on the NTS.
+
+# DATAPATH = "/readonly/repo/main" # NTS - obsolete
+DATAPATH = "/repo/LATISS"  # SUMMIT (warning!)
+# DATAPATH = "/repo/LATISS" # TTS (TBC)
+
+try:
+    butler = dafButler.Butler(
+        DATAPATH,
+        instrument="LATISS",
+        collections=["LATISS/raw/all", "LATISS_test_data"],
+    )
+    DATA_AVAILABLE = True
+except FileNotFoundError:
+    logger.warning("Data unavailable, certain tests will be skipped")

```

Comment:
why does this use `logger.warnings` whereas line 38 uses `warning.warn`

Reply:
I don't know.. I just copied what was in the `test_latiss_cwfs_align.py` file... I think you did that... I actually didn't want to copy and paste it but I didn't find a good way to not duplicate this test... :(

Reply:
Does it matter? I honestly didn't know there were two different "types" of warnings...

Reply:
I think logging and warning have slightly different use-cases.
 
According to the [logging doc page](https://docs.python.org/3/howto/logging.html#when-to-use-logging):

> [warnings.warn()](https://docs.python.org/3/library/warnings.html#warnings.warn) in library code if the issue is avoidable and the client application should be modified to eliminate the warning
>
> [logging.warning()](https://docs.python.org/3/library/logging.html#logging.warning) if there is nothing the client application can do about the situation, but the event should still be noted

I think for this particular one the `warning.warn` is more suitable than the `logging.warning`.

I already modified the code to use `warning.warn`.